app/components/CrashRecommendationCard.tsx

```diff
@@ -0,0 +1,297 @@
+          rec_update: recommendation.rec_update,
+          recommendation_status_id: recommendation.recommendation_status_id,
+          crash_pk: recommendation.crash_pk,
+          recommendations_partners:
+            recommendation.recommendations_partners || [],
+        }
+      : {
+          rec_text: null,
+          rec_update: null,
+          recommendation_status_id: null,
+          crash_pk,
+          created_by: user?.email,
+          recommendations_partners: [],
+        },
+  });
+
+  const { mutate } = useMutation(
+    recommendation
+      ? UPDATE_RECOMMENDATION_MUTATION
+      : INSERT_RECOMMENDATION_MUTATION
+  );
+
+  const onSave = useCallback(
+    async (data: RecommendationFormInputs) => {
+      setIsMutating(true);
+      // just use a generic object to make the data structuring easier
+      const payload: Record<string, unknown> = data;
+
+      const [addPartners, deletePartnerPks] = getPartnerChanges(
+        recommendation?.id || null,
+        recommendation?.recommendations_partners || [],
+        data.recommendations_partners || []
+      );
+
+      let variables;
+      if (recommendation) {
+        delete payload.recommendations_partners;
+        variables = {
+          record: payload,
+          id: recommendation?.id,
+          addPartners,
+          deletePartnerPks,
+        };
+      } else {
+        payload.recommendations_partners = { data: addPartners };
+        variables = { record: payload };
+      }
+      await mutate(variables, { skip_updated_by_setter: true });
+      await onSaveCallback();
+      setIsMutating(false);
+      setIsEditing(false);
+    },
+    [recommendation, mutate, onSaveCallback, setIsMutating, setIsEditing]
+  );
+
+  return (
+    <Card>
+      <Card.Header>Fatality Review Board recommendations</Card.Header>
+      <Card.Body>
+        <Form id="recommendationForm" onSubmit={handleSubmit(onSave)}>
+          <Form.Group className="mb-3">
+            <Form.Label className="fw-bold">Recommendation</Form.Label>
+            {isEditing && (
+              <Form.Control
+                {...register("rec_text", {
+                  // coerce empty fields to null
+                  setValueAs: (v) => v?.trim() || null,
+                })}
+                as="textarea"
+                rows={6}
+                autoFocus={true}
+              />
+            )}
+            {!isEditing && <p>{recommendation?.rec_text || ""}</p>}
+          </Form.Group>
+          <Form.Group className="mb-3">
+            <Form.Label className="fw-bold">Update</Form.Label>
+            {isEditing && (
+              <Form.Control
+                {...register("rec_update", {
+                  // coerce empty fields to null
+                  setValueAs: (v) => v?.trim() || null,
+                })}
+                as="textarea"
+                rows={6}
+              />
+            )}
+            {!isEditing && <p>{recommendation?.rec_update || ""}</p>}
```

Comment:
@johnclary I tested this on a really long collection of updates like in crash `17343328`, and it creates a really tall column. It isn't far from how it looks in production, but the half-width makes it more noticeable. I'm not keen on scrolling since I think the VZ team would want to see the entire contents on screen, but that is only my assumption. I'll think about it some more since I don't have any suggestions on this.

Reply:
whoa—i assumed i had tested a long update, but that's 5k+ characters. thanks for this example. i agree that moving the status + partners field above up top makes sense.

Maybe we should go back to full-width. I think we'd want to set a max container width in that case. After looking more closely at the notes feature, i don't think this layout will work as half-width anyway.

This example also makes me realize that "updates" should actually behave more like notes. It would be cool to take a Moped-style approach and have a note type column that would enable us to re-use the existing notes feature for FRB updates. 

I will get some feedback from Mary on this layout and figure out the best way to move forward in the short term 🙏

Reply:
I moved status + partners to the top of the form. Will open a follow-up issue for further layout tweaks ✅